app-devel/autobuild4/spec

```diff
@@ -1,4 +1,4 @@
 VER=4.2.1
 SRCS="git::commit=tags/v$VER::https://github.com/AOSC-Dev/autobuild4"
 CHKSUMS="SKIP"
-CHKUPDATE="anitya::id=226987"
+CHKUPDATE="github::repo=AOSC-Dev/autobuild4"
```

Comment:
I have just registered an anitya project for autobuild4: https://release-monitoring.org/project/372345/